server.js

```diff
@@ -191,33 +191,39 @@ app.get('/api/upcoming-due-dates', function(req, res) {
   handleAnalyticsCall(req, res, upcomingDueDates);
 });
 
-app.get('/api/sms', function(req, res) {
-  auth.check(req, 'can_access_gateway_api', null, function(err) {
-    if (err) {
-      return serverUtils.error(err, req, res);
-    }
-    smsGateway.get(function(err, obj) {
+(function() {
+  var handleSms_GET = function(req, res) {
```

Comment:
what's up with the underscores? Wouldn't the naming convention / "standard" make these functions `handleSmsGet` and `handleSmsPost`?